src/js/jwplayer.js

```diff
@@ -7,8 +7,8 @@ import { version } from './version';
 import Api from 'api/api';
 import ApiSettings from 'api/api-settings';
 
-/* global __webpack_public_path__:true*/
 /* eslint camelcase: 0 */
+// eslint-disable-next-line
 __webpack_public_path__ = loadFrom();
```

Comment:
eslint didn't like use modifying a "global" here, but really we replace this at build time.